### PR TITLE
Switch to yml issue templates

### PR DESCRIPTION
**Pull request by Basically-Andrew**

### CODE REVIEW

**Comment by KarlOfDuty** [comment]
image

**Comment by Basically-Andrew** [comment]
Switching to yml issue templates allow players less familiar with Github to create an issue using Github's text areas. Its clean and simple for every user to create an issue with this on Github.
image

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Bug Report
+description: Report a bug for the server.
+title: "[Bug]"
+labels: ["Issue type: Bug"]
+body:
+- type: input
+  attributes:
+    label: Server
+    placeholder: "Which server the bug is on."
```
Comment (Warriorrrr) [inline .github/ISSUE_TEMPLATE/bug_report.yml:9]: Don't use placeholder, use description instead. Needs to be done for all fields.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: Feature Request
+description: Suggest a feature, plugin, or mechanic for the server.
+body:
+- type: markdown
+  attributes:
+      value: "Please start your title with one of the following: [Feature], [Plugin], [Mechanic]."
+- type: textarea
+  attributes:
+    label: Is your feature request related to a problem? Please describe.
+    placeholder: "A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]"
```
Comment (Warriorrrr) [inline .github/ISSUE_TEMPLATE/feature_request.yml:10]: Same here

**Review by Warriorrrr — CHANGES_REQUESTED** [review]

**Comment by Basically-Andrew** [comment]
Thanks, for all fields I've moved from placeholder to description instead.

**Review by Warriorrrr — APPROVED** [review]

**Pull request merged**